app/app.go

```diff
@@ -474,10 +475,10 @@ func New(
 		groupmodule.NewAppModule(appCodec, app.GroupKeeper, app.AccountKeeper, app.BankKeeper, interfaceRegistry),
 		evm.NewAppModule(app.EvmKeeper, app.AccountKeeper, evmSs),
 		feemarket.NewAppModule(app.FeeMarketKeeper, feeSs),
-		crosschainmodule.NewAppModule(appCodec, app.ZetaCoreKeeper, app.StakingKeeper, app.AccountKeeper),
-		observermodule.NewAppModule(appCodec, *app.ZetaObserverKeeper, app.AccountKeeper, app.BankKeeper),
-		fungibleModule.NewAppModule(appCodec, app.FungibleKeeper, app.AccountKeeper, app.BankKeeper),
-		emissionsModule.NewAppModule(appCodec, app.EmissionsKeeper, app.AccountKeeper),
+		crosschainmodule.NewAppModule(appCodec, app.ZetaCoreKeeper),
```

Comment:
Not required, it might be a good idea to rename `app.ZetaCoreKeeper` to `app.CrosschainKeeper`

Reply:
Yes, this is correct, we should remove all Zeta prefix and keep `<ModuleName>Keeper` for the convention. For the pointer, I think we can handle it in another PR